roles/github/templates/generic.yml.j2

```diff
@@ -38,8 +47,8 @@ jobs:
         <%- if workflow.arguments is defined +%>
         env:
           KAYOBE_ENVIRONMENT: '%% github_kayobe_arguments.KAYOBE_ENVIRONMENT | default(github_default_kayobe_arguments.KAYOBE_ENVIRONMENT) %%'
```

Comment:
Spotted that by default we use an environment of production, even in the single environment case. Is it possible to not use an environment?

Reply:
The template has been altered to only print `KAYOBE_ENVIRONMENT` if `github_environment_selector:` is not null/none.